### PR TITLE
Textarea widget displaying recording wall clock time in OSD

### PR DESCRIPTION
**Pull request by mpaolino**
Ticket: https://code.mythtv.org/trac/ticket/12969#ticket

This adds a textarea widget named "wallclocktime" to the OSD that will display the local playing wall clock time of the recording being played. Its very useful to me, don't if others will want it. I've being using it for 3 week without issues.

Example snippet from my osd.xml theme file, moving the description text widget down and showing the recording time in its place.


```xml
       <textarea name="wallclocktime">                                                                                                                                                                                                
            <font>small</font>
            <area>10,5,1700,40</area>
            <align>hcenter, bottom</align>
            <template>Wall clock: %1</template>
        </textarea> 
        <textarea name="description">
            <font>small</font>
            <area>10,70,1700,40</area>
            <align>hcenter,bottom</align>
            <template>%DESCRIPTION%
            %(|REMAININGTIME|)%%(|BEHINDTIME|)%%VALUE%%UNITS%</template>
        </textarea> 
```

### CODE REVIEW

File: mythtv/libs/libmythtv/osd.cpp [hunk 1]
```diff
@@ -542,6 +542,13 @@ void OSD::SetText(const QString &window, const InfoMap &map,
         bar->SetStart(0);
         bar->SetTotal(1000);
     }
+ 
+	if (map.contains("wallclock"))
```
Comment (mpaolino) [inline mythtv/libs/libmythtv/osd.cpp]: This change is wrong, updating.

**Comment by twitham1** [comment]
I've often wondered what time it was that I am watching.  Turns out you figured this out 4 years ago, thanks!

My fork now has 2 commits that work against current fixes/31 (tested) and master (untested).  I also added the date to the recording clock, and added it to the three themes in the source.  I actually use this with the Mythbuntu theme though.

**Comment by twitham1** [comment]
My master fork now compiles but I can't run it against my fixes/31 mythtv.

Rather than the fixed format, it might be better to honor the configured date/time formats.  But I have not yet attempted this.

**Comment by mpaolino** [comment]
@twitham1 sorry mate, can't really help to sort this out as I'm not using this anymore. But I'm glad you found it useful, I hope you can pick up from here. Best.

**Comment by twitham1** [comment]
Thank you for the reply!  In that case, I think this one can be closed and replaced by mine.  Done.

**Comment by linuxdude42** [comment]
Replaced by #322.

**Pull request closed without merging**